azurelinuxagent/daemon/main.py

```diff
@@ -149,9 +148,6 @@ def daemon(self, child_args=None):
         # that require the goal state and IMDS
         self._initialize_telemetry()
 
-        # Initialize the agent cgroup
```

Comment:
I am doing all the cgroups stuff in the extension handler. Systemd does all the setup and the handler simply discovers the setup and starts tracking the cgroups. When I get to non-systemd distros I will have to do some of the setup in the daemon, but for now it is not needed.